src/cache_invalidate.c

```diff
@@ -101,6 +102,21 @@ cache_invalidate_callback(Datum arg, Oid relid)
 	}
 }
 
+static void
+cache_invalidate_callback(Datum arg, Oid relid)
+{
+	/* original callback */
+	cache_invalidate_callback_internal(arg, relid);
```

Comment:
We have just renamed this function and are still calling it unconditionally. So I fail to see how this is solving the problem at hand.